blog/2024-08-27-entity-service-schema-validation.md

```diff
@@ -0,0 +1,15 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Validation of entity action schemas"
+---
+
+It's no longer possible to register an entity action  with a custom schema which does not meet at least one of these criteria:
+- A validator returned by `cv.make_entity_service_schema`
+- A validator returned by `cv.make_entity_service_schema`, wrapped in a `vol.Schema`
+- A validator returned by `cv.make_entity_service_schema`, wrapped in a `vol.All`
+
+It's still possible to register an entity action with no schema, or with a dict representing the additional action call parameters.
+The reason for the change is that if `cv.make_entity_service_schema` is not used, the service will not automatically support all possible ways of targeting entities.
+
+More details can be found in core [PR #124102](https://github.com/home-assistant/core/pull/124102)
```

Comment:
Instead we should update the docs page section with the new schema requirements and link to that page anchor.

https://developers.home-assistant.io/docs/dev_101_services#entity-service-actions

Reply:
I added a link to the documentation. I want to keep the link to the core PR for context.

Reply:
Linking to a PR is a habit that will allow us to skip linking to a docs page which isn't good.

Reply:
We should update the docs page too.

Reply:
> Linking to a PR is a habit that will allow us to skip linking to a docs page which isn't good.

Maybe, but now the blog post links to both

> We should update the docs page too

That's already been done.

Can we please get this PR merged, as custom integration authors need to know what's going on?